data/en/createguid.json

```diff
@@ -7,7 +7,8 @@
 	"description": "Creates a globally unique identifier (32 character hexadecimal string)",
 	"params": [],
 	"engines": {
-		"lucee": {"minimum_version": "", "notes": "", "docs": "https://docs.lucee.org/reference/functions/createguid.html"}
+		"lucee": {"minimum_version": "", "notes": "", "docs": "https://docs.lucee.org/reference/functions/createguid.html"},
+		"railo": {"minimum_version": "", "notes": "", "docs": "http://www.crala.org/railo-context/doc/functions.cfm?item=createguid"}
```

Comment:
I don't want to link to some random server's open railo-context for the docs here.

Reply:
@pfreitag got it. Cancelling this pull request.